internal_testsuites/golang/test_helpers/enclave_setup.go

```diff
@@ -32,10 +37,20 @@ func CreateEnclave(t *testing.T, ctx context.Context, testName string) (resultEn
 
 	}
 	destroyEnclaveFuncWrapped := func() error {
-		if err := destroyEnclaveFunc(); err != nil {
-			logrus.Errorf("An error occurred destroying enclave '%v' that we created for this test:\n%v", enclaveName, err)
-			logrus.Errorf("ACTION REQUIRED: You'll need to destroy enclave '%v' manually!!!!", enclaveName)
-			return err
+		var i uint32
+		for i = uint32(0); i < destroyEnclaveRetries; i++ {
```

Comment:
perhaps we can do `i := 0` ? if we aren't using this index elsewhere

Reply:
I forgot to change that after i moved the end check inside the loop.